UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Objects.Security;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Systems
+{
+	/// <summary>
+	/// Contains a list of all mainstation crewmembers, including traitors and silicons but not, for example, fugitives or wizards.
+	/// </summary>
+	public class CrewManifestManager : MonoBehaviour
+	{
+		public static CrewManifestManager Instance;
```

Comment:
Codacy found an issue: [Change the visibility of 'Instance' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)